config/nginx.conf.erb

```diff
@@ -24,6 +24,8 @@ http {
 	default_type application/octet-stream;
 	sendfile on;
 
+	resolver 8.8.8.8 valid=10s;
```

Comment:
I figure if we use cloudflare already, we might as well use their DNS servers (`1.1.1.1`) to have the most up-to-date info, no?